pkg/metrics/dapr_metrics.go

```diff
@@ -0,0 +1,95 @@
+package metrics
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"contrib.go.opencensus.io/exporter/prometheus"
+	ocprom "contrib.go.opencensus.io/exporter/prometheus"
```

Comment:
ocprom is already imported in above line...

Reply:
oh nice catch! :)